user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Changing_the_IP_of_a_DMA.md

```diff
@@ -43,7 +43,7 @@ For a standalone DMA, i.e. a DMA that is not combined with other DMAs in a clust
 
    1. Restart the elasticsearch-service-x64 service.
 
-1. If the server has NATS installed:
+1. If the server has the NATS service installed:
```

Comment:
```suggestion
1. If the server has NATS installed:
```

Can we leave this as "NATS" or does that change the content of the sentence, @lustpieter ?

Reply:
"the NATS service" is more complete, just saying NATS may be confusing

Reply:
Okay, thanks! :)